native/cocos/engine/BaseEngine.h

```diff
@@ -104,11 +104,11 @@ class CC_DLL BaseEngine : public CallbacksInvoker,
      */
     virtual void removeEventCallback(OSEventType evtype) = 0;
 
-    using SchedulerPtr = std::shared_ptr<Scheduler>;
+    using SchedulerSharePtr = std::shared_ptr<Scheduler>;
```

Comment:
Why rename this?

Reply:
Because ptr is too abstract. The specific points may be better understood